platform.md

```diff
@@ -122,6 +137,11 @@ The buildpacks directory MUST contain unarchived buildpacks such that:
 - Each top-level directory is a buildpack ID.
 - Each second-level directory is a buildpack version.
 
+## OCI Image
```

Comment:
Calling this the `OCI Image` implies to me the the other described images are somehow not OCI images which is untrue. In the past we have informally referred to this as the App Image or the Exported Image. Maybe we can use one of those or come up with a new idea.

Reply:
Thanks @ekcasey ; we've renamed this to Exported Image.